### PR TITLE
Add pandas_load_args for LoadCSV

### PR DESCRIPTION
**Pull request by elronbandel**
Closes: https://github.com/IBM/unitxt/issues/695

### CODE REVIEW

File: tests/library/test_loaders.py [hunk 1]
```diff
@@ -68,6 +68,28 @@ def test_load_csv(self):
...
                     self.assertEqual(saved_instance[1].to_dict(), loaded_instance)
 
+    def test_load_csv_with_pandas_args(self):
+        # Using a context for the temporary directory
+        with tempfile.TemporaryDirectory() as tmp_dir:
+            files = {}
+            dfs = {}
+
+            for file in ["train", "test"]:
+                path = os.path.join(tmp_dir, file + ".csv")  # Adding a file extension
```
Comment (benjaminsznajder) [inline tests/library/test_loaders.py:78]: not critical, but makes sense to call it tsv.

**Review by benjaminsznajder — COMMENTED** [review]
It works. But I have a general comment.

The API by requesting optional pandas_load_args argument makes the assumption that the implementation is by pd.read_csv()
The user can not make this assumption... shouldn't we just give the 'sep' argument?

File: tests/library/test_loaders.py (hunk 1 above)
Comment (elronbandel) [reply tests/library/test_loaders.py]: done

**Comment by elronbandel** [comment]
Done.

**Review by benjaminsznajder — APPROVED** [review]

**Pull request merged**